homeassistant/components/logbook.py

```diff
@@ -93,12 +90,7 @@ class LogbookView(HomeAssistantView):
 
     def get(self, request, date=None):
```

Comment:
This is no longer a date so should not be called like that. We should also update the `extra_urls` to reflect this.